source/features/faster-reviews.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import onReplacedElement from '../helpers/on-replaced-element';
+
+async function addSidebarReviewButton(): Promise<void> {
+	const reviewFormUrl = new URL(location.href);
+	reviewFormUrl.pathname += '/files';
+	reviewFormUrl.hash = 'submit-review';
+
+	const sidebarReviewsSection = await elementReady('[aria-label="Select reviewers"] .discussion-sidebar-heading');
+	sidebarReviewsSection!.append(
+		<span style={{fontWeight: 'normal'}}>
+			– <a href={reviewFormUrl.href} className="btn-link muted-link" data-hotkey="v">review</a>
```

Comment:
![image](https://user-images.githubusercontent.com/16872793/83874958-49e9fa80-a704-11ea-8c4b-de6c804736cd.png)

I think it looks a bit odd. Maybe it should say `Submit a review`?

Reply:
It matches the style of `Assignees – assign yourself`

Reply:
Could be change to “review now” to make it clear it’s an action